src/App.js

```diff
@@ -1,25 +1,135 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import "./App.css";
+
+import React, { Component } from "react";
+import GeneralInfo from "./components/GeneralInfo";
+import Education from "./components/Education";
+import Experience from "./components/Experience";
+import CVPreview from "./components/CVPreview";
+import Skills from "./components/Skills";
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      generalInfo: {
+        name: "",
+        email: "",
+        phone: "",
+        address: "",
+        description: "",
+      },
+      education: [],
+      experience: [],
+      skills: [],
+    };
+  }
+
+  handleGeneralInfoSubmit = (name, email, phone, address, description) => {
```

Comment:
can we user `object restructuring`  here ?

Reply:
please check the same at other place as well like for function -- `handleEducationSubmit`